### DISCUSSION TITLE
How should i choose window_size_samples?

### DISCUSSION THREAD

**Discussion by wl-junlin** [snakers4/silero-vad#324] [Q&A] [answered]
it was said in the comment "Silero VAD models were trained using 512, 1024, 1536 samples for 16000 sample rate"
so, for a better acuuracy, should i chosse 1536 as my window_size_samples?
however, for a better lantancy, i should choose 512?

**Comment by snakers4** [comment, marked as answer]
The bigger the window size, the higher the quality.
With an obvious latency trade off.

**Reply by snakers4** [reply]
For v4 we did not publish the quality charts comparing 512 vs 1576, but the bigger the better.
New model has a smaller quality difference than older models, though.
